shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/KonanTarget.kt

```diff
@@ -188,23 +192,37 @@ open class HostManager(protected val distribution: Distribution = Distribution()
                     // TODO: toolchain to be fixed for that to work.
                     // KonanTarget.ANDROID_ARM64,
                     KonanTarget.WASM32
-                ) + zephyrSubtargets
+                )
                 KonanTarget.MACOS_X64 -> listOf(
                     KonanTarget.MACOS_X64,
                     KonanTarget.IOS_ARM32,
                     KonanTarget.IOS_ARM64,
                     KonanTarget.IOS_X64,
                     KonanTarget.LINUX_X64,
                     KonanTarget.LINUX_ARM32_HFP,
+                    KonanTarget.MINGW_X64,
                     KonanTarget.ANDROID_ARM32,
                     KonanTarget.ANDROID_ARM64,
                     KonanTarget.WASM32
-                ) + zephyrSubtargets
-                else ->
-                    throw TargetSupportException("Unknown host platform: $host")
+                )
+                else -> throw TargetSupportException("Unknown host platform: $host")
             }        
     }
 
+    val enabledExperimental: List<KonanTarget> by lazy {
```

Comment:
It isn't really enabled - zephyr and mingw on linux did not built by `./gradlew clean build bundle`

Reply:
Sure, it's WIP PR.